tst/EnergyPlus/unit/FanCoilUnits.unit.cc

```diff
@@ -1,4375 +0,0 @@
-// EnergyPlus, Copyright (c) 1996-2020, The Board of Trustees of the University of Illinois,
```

Comment:
Any reason why this whole file got deleted?  Or something?

Reply:
I did resolve unit tests conflict and tested locally and every thing was fine. May be I did something inadvertently. I will fix that.